src/uebung_3_3/BMIRechner.java

```diff
@@ -0,0 +1,46 @@
+package uebung_3_3;
+
+public class BMIRechner {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		double Alter = 23;
+		double Gewicht = 71;
+		double Groesse = 1.65;
+		boolean Maennlich = false;
+		double BMI = Gewicht / (Groesse * Groesse);
+		System.out.println("BMI " + BMI);
+
+		if (Maennlich) {

```

Comment:
Hervorragend, Sie haben einen sinnvollen Nutzen für `^` gefunden! Ist mir nie in den Sinn gekommen das so zu lösen! 🏆